certified-connectors/Ubiqod by Skiply/apiDefinition.swagger.json

```diff
@@ -0,0 +1,273 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Ubiqod by Skiply",
+    "description": "Use Ubiqod connected buttons to trigger your flows.",
```

Comment:
Could the description please be expanded a bit more? Things like what the underlying service is, what value the connector offers, etc. are helpful to include here. 

As an aside, friendly reminder of the 500 character limit for the description field.

Thanks in advance!

Reply:
Hi Doug,
Thank you for your review. I just pushed a new description.
Cheers,
Jérome